tools/generators/star-dev-deps/index.spec.ts

```diff
@@ -0,0 +1,160 @@
+import { createTreeWithEmptyWorkspace } from '@nrwl/devkit/testing';
+import {
+  Tree,
+  readProjectConfiguration,
+  addProjectConfiguration,
+  serializeJson,
+  readWorkspaceConfiguration,
+  readJson,
+} from '@nrwl/devkit';
+
+import generator from './index';
+import { StarDevDepsGeneratorSchema } from './schema';
+
+describe('star-dev-deps generator', () => {
```

Comment:
Should we also test that a non-internal package is not affected?

Reply:
already covered by `should not modify external dev dependencies`

Reply:
Somehow completely missed that test. Disregard